projects/jupyter-server-ydoc/jupyter_server_ydoc/loaders.py

```diff
@@ -117,7 +117,7 @@ async def load_content(self, format: str, file_type: str) -> dict[str, Any]:
             self.last_modified = model["last_modified"]
             return model
 
-    async def maybe_save_content(self, model: dict[str, Any]) -> None:
+    async def maybe_save_content(self, model: dict[str, Any]) -> dict[str, Any] | None:
```

Comment:
It looks like this method never returns `None`?
```suggestion
    async def maybe_save_content(self, model: dict[str, Any]) -> dict[str, Any]:
```

Reply:
I think it does return `None` - if it catches `asyncio.CancelledError` in lines 155-156, then it returns `saved_model = None` as defined in line 152.

Reply:
Oh right, my bad.